### ISSUE TITLE
Applications say the build successfully in the log but don't execute properly. 

### ISSUE THREAD

**Issue by webhead404** [frikky/Shuffle#151]
**Describe the bug**
After a clean build of Shuffle some of the older apps do not build correctly. 

**To Reproduce**
1. Run installation per the docs
2. Download apps from Github button - frikky/shuffle-apps
3. Create a workflow
4. Execute an action in the workflow (Testing, Http, etc.)

**Expected behavior**
Use the HTTP app and curl google.com. Result should be Success, and the output of the curl command representing an http request to google.com. 

**Actual result**
Shuffle docker containers build fine, but applications that were tested did not produce a result. Execution was in a hung state. 

**System info**
Currently testing on a Ubuntu 18 VM with 200 GB of RAM. Installed Docker CE and Docker Compose via the official Docker documentation.

**Comment by frikky** [comment]
[message.txt](https://github.com/frikky/Shuffle/files/5226656/message.txt)

Thaaaank you! Attaching the logs you sent as well. 
TL;DR for self: fails to build apps for some reason. Related to downloading [these alpine packages](http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz)

**Comment by webhead404** [comment]
Okay I think the downloading piece was DNS related. However the workflows still don't execute fully. 

```2020/09/15 17:51:54 [INFO] Starting execution of f59df672-9cad-4013-b742-afbc8146fb9d!
2020/09/15 17:51:54 [INFO] Should start execution on node aa0e71e6-838c-4415-8b23-dae151d56bcb
2020/09/15 17:51:54 [INFO] No execution source (trigger) specified. Setting to default
2020/09/15 17:51:54 [INFO] New startnode: aa0e71e6-838c-4415-8b23-dae151d56bcb
2020/09/15 17:51:54 [FIXME] ImageNames to check: []string{"frikky/shuffle:http_1.0.0"}
2020/09/15 17:51:54 [INFO] Execution: 9812867d-ff4e-4fcc-a336-459d699ae059 should execute onprem with execution environment "Shuffle"
2020/09/15 17:51:54 [INFO] Executionrequests: 1
2020/09/15 17:51:55 No users found for apikey d413a775-6260-4e7f-8e54-04b1504f116f
2020/09/15 17:51:55 Api authentication failed in abort workflow: User is invalid - no username found```

**Comment by webhead404** [comment]
Closing this, appears to have been a network issue. That was fixed by Forcing the update rather than uploading from GitHub.
